src/pytkdocs/objects.py

```diff
@@ -350,6 +350,18 @@ class Class(Object):
 
     possible_name_properties: List[ApplicableNameProperty] = [NAME_PRIVATE]
 
+    def __init__(self, *args, bases: List[str] = ["object"], **kwargs):
```

Comment:
Just saw this (not sure it was there earlier, sorry for missing it): I don't think it has any impact on the current code, but mutable defaults are not considered good-practice.
If `bases` is not specified when creating instances of `Class`, all these instances will share the same `["object"]` list.
Could you instead default to None and create the `["object"]` list in the body of the function?

Reply:
Ah, yeah, good idea